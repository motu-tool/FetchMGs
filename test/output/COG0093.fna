version https://git-lfs.github.com/spec/v1
oid sha256:b888bf0c208c1b38ec506f534b0718b1817462a51029df05f311be00b2156563
size 2555
