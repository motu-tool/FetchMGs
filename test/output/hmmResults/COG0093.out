# hmmsearch :: search profile(s) against a sequence database
# HMMER 3.3.2 (Nov 2020); http://hmmer.org/
# Copyright (C) 2020 Howard Hughes Medical Institute.
# Freely distributed under the BSD open source license.
# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
# query HMM file:                  /nfs/home/fieldc/.local/lib/python3.7/site-packages/fetchmgs/data/COG0093.hmm
# target sequence database:        ../fetchMGs/example_datasets/example_data_genomes.faa
# output directed to file:         output/hmmResults/COG0093.out
# per-dom hits tabular output:     output/hmmResults/COG0093.dom
# show alignments in output:       no
# max ASCII text line length:      unlimited
# sequence reporting threshold:    score >= 80
# number of worker threads:        1
# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -

Query:       COG0093  [M=132]
Scores for complete sequences (score includes all domains):
   --- full sequence ---   --- best 1 domain ---    -#dom-
    E-value  score  bias    E-value  score  bias    exp  N  Sequence           Description
    ------- ------ -----    ------- ------ -----   ---- --  --------           -----------
    3.7e-65  217.9   2.6    4.1e-65  217.8   2.6    1.0  1  1089456.NCGM2_5498  protein_id="BAK92319.1" gene="rplN" product="50S ribosomal protein L14" GI="348036959" note="similar to GI:116052288; similar to PA14_08950 of P. aeruginosa UCBPP-PA14" project_id="73453"
    8.3e-65  216.8   5.2    9.1e-65  216.7   5.2    1.0  1  1037409.BJ6T_43970  protein_id="BAL09666.1" gene="rplN" product="50S ribosomal protein L14" GI="354956987" project_id="67463"
      2e-62  209.1   3.1    2.2e-62  208.9   3.1    1.0  1  944546.ABED_0709    protein_id="BAK70426.1" product="50S ribosomal protein L14" GI="345468975" project_id="61719"
    2.4e-62  208.9   2.8    2.6e-62  208.7   2.8    1.0  1  944547.ABLL_0984    protein_id="BAK72859.1" product="50S ribosomal protein L14" GI="345471409" project_id="61721"
    1.5e-55  186.8   0.3    1.7e-55  186.7   0.3    1.0  1  1148.SYNGTS_0765    protein_id="BAK49513.1" gene="rpl14" product="50S ribosomal protein L14" GI="339273026" note="ORF_ID:sll1806" project_id="67081"


Domain annotation for each sequence:
>> 1089456.NCGM2_5498  protein_id="BAK92319.1" gene="rplN" product="50S ribosomal protein L14" GI="348036959" note="similar to GI:116052288; similar to PA14_08950 of P. aeruginosa UCBPP-PA14" project_id="73453"
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 !  217.8   2.6   8.7e-69   4.1e-65      11     132 .]       1     122 []       1     122 [] 0.99

>> 1037409.BJ6T_43970  protein_id="BAL09666.1" gene="rplN" product="50S ribosomal protein L14" GI="354956987" project_id="67463"
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 !  216.7   5.2     2e-68   9.1e-65      11     132 .]       1     122 []       1     122 [] 0.99

>> 944546.ABED_0709  protein_id="BAK70426.1" product="50S ribosomal protein L14" GI="345468975" project_id="61719"
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 !  208.9   3.1   4.8e-66   2.2e-62      11     132 .]       1     122 []       1     122 [] 0.99

>> 944547.ABLL_0984  protein_id="BAK72859.1" product="50S ribosomal protein L14" GI="345471409" project_id="61721"
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 !  208.7   2.8   5.6e-66   2.6e-62      11     132 .]       1     122 []       1     122 [] 0.99

>> 1148.SYNGTS_0765  protein_id="BAK49513.1" gene="rpl14" product="50S ribosomal protein L14" GI="339273026" note="ORF_ID:sll1806" project_id="67081"
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 !  186.7   0.3   3.6e-59   1.7e-55      11     132 .]       1     122 []       1     122 [] 0.99



Internal pipeline statistics summary:
-------------------------------------
Query model(s):                            1  (132 nodes)
Target sequences:                      23272  (7230323 residues searched)
Passed MSV filter:                       493  (0.0211843); expected 465.4 (0.02)
Passed bias filter:                      374  (0.0160708); expected 465.4 (0.02)
Passed Vit filter:                        37  (0.00158989); expected 23.3 (0.001)
Passed Fwd filter:                         7  (0.000300791); expected 0.2 (1e-05)
Initial search space (Z):              23272  [actual number of targets]
Domain search space  (domZ):               5  [number of targets reported over threshold]
# CPU time: 0.10u 0.01s 00:00:00.11 Elapsed: 00:00:00.07
# Mc/sec: 13357.42
//
[ok]
